icu4j/main/classes/core/src/com/ibm/icu/message2/MessageFormatter.java

```diff
@@ -0,0 +1,333 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2;
+
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * {@code MessageFormatter} is the next iteration of {@link com.ibm.icu.text.MessageFormat}.
+ *
+ * <p>For more than two years the Message Format Working Group (MFWG) worked on
```

Comment:
The discussion of the background should not talk about the MFWG, but rather just focus on the feature comparison to MF1.0

Reply:
Done.